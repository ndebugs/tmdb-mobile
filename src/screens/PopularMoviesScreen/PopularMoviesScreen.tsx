import React from 'react';
import {
  View,
  Text,
} from 'react-native';
import { Container, Header } from '../../components';

const PopularMoviesScreen = () => {
  return (
    <Container>
      <Header title="Movies" />
      <View style={{ flex: 1, alignItems: 'center', justifyContent: 'center' }}>
        <Text testID="blankLabel">This is Popular Movies screen.</Text>
      </View>
    </Container>
  );
};

export default PopularMoviesScreen;
