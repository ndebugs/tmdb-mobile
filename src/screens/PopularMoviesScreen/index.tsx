import PopularMoviesScreen from './PopularMoviesScreen';

export default PopularMoviesScreen;
