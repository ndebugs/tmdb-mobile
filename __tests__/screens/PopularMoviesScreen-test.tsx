import React from 'react';
import * as redux from 'react-redux';
import { shallow } from 'enzyme';
import PopularMoviesScreen from '../../src/screens/PopularMoviesScreen';

jest.mock("react-redux", () => ({
  useSelector: jest.fn(fn => fn()),
  useDispatch: () => jest.fn()
}));

const spy = jest.spyOn(redux, 'useSelector');
spy.mockReturnValue({});

describe('<PopularMoviesScreen>', () => {
  it('should match snapshot', () => {
    const component = shallow(<PopularMoviesScreen />);
    expect(component).toMatchSnapshot();
  });
});
