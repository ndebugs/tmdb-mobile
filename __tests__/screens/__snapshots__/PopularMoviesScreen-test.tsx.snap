// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<PopularMoviesScreen> should match snapshot 1`] = `ShallowWrapper {}`;
